FeedStoreChallenge/Entities/ManagedCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ManagedCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Istvan Private on 14.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
+		return NSFetchRequest<ManagedCache>(entityName: "ManagedCache")
+	}
+
+	@NSManaged public var timestamp: Date
+	@NSManaged private var feedImagesOrderedSet: NSOrderedSet
+
+	public var feedImages: [ManagedFeedImage] {
+		get { feedImagesOrderedSet.array as? [ManagedFeedImage] ?? [] }
```

Comment:
A better approach when dealing with a possible heterogeneous `NSOrderedSet` would be to use a `compactMap`, so all objects that can be downcasted to `ManagedFeedImage` are returned instead of an empty array in case of failure from a single one.
```suggestion
		get { feedImagesOrderedSet.compactMap { $0 as? ManagedFeedImage } }
```